app/templates/frameworks/_framework_actions.html

```diff
@@ -35,8 +35,7 @@
     </a>
   {% endif %}
   <p class="browse-list-item-body">
-    Agree to the terms of the bid, provide supplier information and
-    confirm&nbsp;eligibility.
+    Confirm eligibility, agree to the application terms and provide supplier information.
```

Comment:
I'm only guessing this was the reason before - I personally don't have a problem with this.